src/forms/numberInput.test.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import NumberInput, {DECREMENT_BTN_CLASS, INCREMENT_BTN_CLASS} from './NumberInput';
+
+describe('NumberInput', function() {
```

Comment:
Test we should add,
- calling increment and it updating state with expected value
- calling decrement and it updating state with expected value
- calling onFocus affect on classList
- calling onChange and it updating state with expected value

To get access to the instance so you can use 
`const numberInput = TestUtils. findRenderedComponentWithType(numberInputComponent, NumberInput);`